src/performance_profiles.jl

```diff
@@ -13,21 +13,22 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
 
   (np, ns) = size(T);       # Number of problems and number of solvers.
 
-  T[isinf(T)] = NaN;
-  T[T .< 0] = NaN;
+  T[isinf.(T)] = Inf;
+  T[T .< 0] = Inf;
   minperf = minimum(T, 2);  # Minimal (i.e., best) performance per solver
 
   # Compute ratios and divide by smallest element in each row.
   r = zeros(np, ns);
   for p = 1 : np
     r[p, :] = T[p, :] / minperf[p];
   end
+  r[isinf.(r)]=NaN
 
-  logscale && (r = log2(r));
-  max_ratio = maximum(r);
+  logscale && (r = log2.(r));
+  max_ratio = findmax(r)[1]
```

Comment:
I think that's a bug in Julia. See
* https://github.com/JuliaLang/julia/issues/22337 and
* https://discourse.julialang.org/t/inconsistency-between-findmax-and-maximum-with-respect-to-nan/4214/3.

It's time to use [NaNMath.jl](https://github.com/mlubin/NaNMath.jl):
```julia
julia> NaNMath.max(2, NaN)
2.0

julia> NaNMath.maximum([2, NaN])
2.0
```